presto-spi/src/main/java/com/facebook/presto/spi/ColumnMetadata.java

```diff
@@ -31,23 +31,30 @@
     private final String extraInfo;
     private final boolean hidden;
     private final Map<String, Object> properties;
+    private final Object defaultValue;
```

Comment:
why not `Optional`?

Reply:
One interesting question is if you can represent `DEFAULT NULL`. It seems like we should in the AST, since it's generally best for the AST to round trip (although we have not been diligent about that historically). It seems less useful from a metadata perspective.